### ISSUE TITLE
1.2.17: CS0006: Metadata file '<extension folder>\Analyzers\MDKAnalyzer.dll' could not be found.

### ISSUE THREAD

**Issue by Auhrii** [malware-dev/MDK-SE#153]
Clean/rebuild or attempting to deploy any solution after updating to 1.2.17 yields an error claiming MDKAnalyzer.dll does not exist - further inspection found the extension folder didn't exist, but a differently named one containing MDK files (including the Analyzer) did. Manually updating the MDK.paths.props file to point to the new folder fixed the issue.

Attached is one of my solution folders with an MDK subfolder. Please accept this sacrifice to the almighty gods of C̵̮̳͇̊̇͊͋̀̀͛́̄̂́́̕͠#̷̧͔̦͖̮͔̣̄̇̈́̿̃̕̚͠.

[Rover Life Support Controller.zip](https://github.com/malware-dev/MDK-SE/files/3273410/Rover.Life.Support.Controller.zip)

**Comment by cjbehm** [comment]
I also ran into this; and after modifying my .csproj file to point to the correct location, started getting errors about the netstandard and the Analyzer references.
Adding <Reference Include="netstandard" /> did not fix it until after I deleted the .vs folder. Then opening the project prompted some sort of repair message, which did fix everything up to the point that Deploy works again.

**Comment by cjbehm** [comment]
One of the things that definitely changed was that files inside the MDK directory in the solution changed from mdk.options to mdk.options.props and mdk.paths.props. So definitely some sort of significant change between whatever version I was on from the last time I went in and updated my scripts and now.

**Comment by malware-dev** [comment]
Yes, that's how I made it support code repositories with local paths.

**Comment by cjbehm** [comment]
I'm sure that's true; was just pointing out that those were all tied to what I had to manually resolve.

After the new version had installed (into Visual Studio 2017; Community), the errors window said it couldn't find the Analyzer. After fixing that manually in csproj, it then complained that it couldn't open mdk.options.prop (which didn't exist).

After I deleted the .vs directory, I was prompted about a repair and that's what finally seemed to have fixed everything.
